util/xmlparser.py

```diff
@@ -78,44 +48,20 @@ def int_of_optional_subtag(self, tag, name, namespaces=None):
             return v
         return int(v)
 
-    def scrub_xml(self, xml):
```

Comment:
"Scrub" is the xml term for cleaning the data, right?

Reply:
It's not official but yeah, it refers to taking some data that might contain invalid characters, and removing those characters.